### ISSUE TITLE
OutputSettings Exceeded Error

### ISSUE THREAD

**Issue by TylerTCF** [pyviz/holoviews#2993]
Hi all,

Is there a way to get past this issue?

```
ds=hv.Dataset(temp3.filter(['business_line_function_name', 'site_name', 'Year', 'Things']))
ds.to(hv.HeatMap, ['Year', 'business_line_function_name'], ['site_name'], ['Things'])

temp3.shape is (16522, 12)

Skipping regular visual display to avoid lengthy animation render times
[Total item frames exceeds max_frames on OutputSettings (500)]
:HoloMap   [Things]
   :HeatMap   [Year,business_line_function_name]   (site_name)
```

Thanks!

**Comment by jlstevens** [comment]
If you are using the notebook, you can use the ``%output`` magic to specify a larger value for `max_frames`. That warning is there for a reason though...if you are using matplotlib it might take a while to render all your data!

**Comment by philippjfr** [comment]
That said the warning should describe exactly how to increase the limit.

**Comment by jlstevens** [comment]
Agreed.

**Comment by TylerTCF** [comment]
Apologize, I am clearly less familiar with the cell/line magic than I should be.  It was introduced to me by James at the last AnacondaCON when I learned about holoviews.  I appreciate you telling me how to override it, but it looks like I found a reasonable workaround using hvplot now and doesn't require me to change the settings.  Is there a place you recommend that I can read up on this topic?  Many thanks.
